service/controller/resource/spark/testdata/simple_machinepool.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: exp.cluster.x-k8s.io/v1alpha3
+kind: MachinePool
+metadata:
+  labels:
+    azure-operator.giantswarm.io/version: 4.2.0
+    cluster.x-k8s.io/cluster-name: c6fme
+    giantswarm.io/machine-pool: nopo1
+    giantswarm.io/cluster: c6fme
+    giantswarm.io/organization: giantswarm
+    release.giantswarm.io/version: 12.0.0
+  name: nopo1
+  namespace: default
+spec:
+  clusterName: c6fme
+  failureDomains:
+    - "1"
+    - "3"
+  replicas: 2
+  template:
+    metadata: { }
+    spec:
+      bootstrap:
+        configRef:
+          apiVersion: bootstrap.cluster.x-k8s.io/v1alpha3
+          kind: KubeadmConfig
```

Comment:
:question:

Reply:
I just copied it from somewhere else. I'll remove that part.